README.md

```diff
@@ -129,7 +129,7 @@ by executing the following command:
   docker-compose up
 ```
 
-// The port should be 2704, shouldn't it?
+// The port should be 3704, shouldn't it?
```

Comment:
Why not updating the below points of the readme and deleting the comment? :)

Reply:
updated ! ;)